src/spec.ts

```diff
@@ -50,8 +50,27 @@ export interface FacetSpec extends BaseSpec {
   spec: LayerSpec | UnitSpec;
 }
 
+export enum ResolveStrategy {
```

Comment:
ResolveMode?  

Strategy generally refer to higher-level picture.  This is a very low level detail.